e2e/test/iothub/DeviceClientX509AuthenticationE2ETests.cs

```diff
@@ -71,12 +58,6 @@ public async Task X509_InvalidDeviceId_Throw_UnauthorizedException_Mqtt_WebSocke
             await X509InvalidDeviceIdOpenAsyncTest(DeviceTransportType.Mqtt_WebSocket_Only).ConfigureAwait(false);
         }
 
-        [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
-        public async Task X509_InvalidDeviceId_Throw_UnauthorizedException_Twice_Amqp()
-        {
-            await X509InvalidDeviceIdOpenAsyncTwiceTest(DeviceTransportType.Amqp).ConfigureAwait(false);
-        }
-
         [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
         public async Task X509_InvalidDeviceId_Throw_UnauthorizedException_Twice_Amqp_TCP()
```

Comment:
What's the point of having another test that does it twice? Was there some bug where it wouldn't throw on a second try?

Seems like we should delete the "twice" tests, or remove the singles and just have the doubles.

@timtay-microsoft @abhipsaMisra ?

Reply:
I'm not finding any additional context in the PR associated with this change: https://github.com/Azure/azure-iot-sdk-csharp/pull/1136. It looks like this manual check was added in as a part of a bug fix (#1137 , #1130) but I don't have the context on what the bug/fix was.
In the absence of any additional context, having the "single run" test looks good to me.